src/components/Logo.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import styled from 'styled-components';
+import logo from 'open-city-design/src/assets/helsinki-logo-black.svg';
```

Comment:
We already talked in the morning, that at some point we should switch into: https://github.com/airbnb/babel-plugin-inline-react-svg to have better control over the SVG's